orchagent/dash/dashrouteorch.cpp

```diff
@@ -112,6 +115,17 @@ bool DashRouteOrch::addOutboundRouting(const string& key, OutboundRoutingBulkCon
         }
         outbound_routing_attrs.push_back(outbound_routing_attr);
     }
+    else if (ctxt.metadata.routing_type() == dash::route_type::RoutingType::ROUTING_TYPE_PRIVATELINK
```

Comment:
The HLD specifically calls out the scenario where routing_type is privatelink, do we need to update the HLD?

Reply:
@prsunny please confirm that the route entry cannot have routing_type directly as PL here. 

`underlay_sip             = ip_address                ; underlay ipv4 src ip if routing_type is {servicetunnel,privatelink}; this is the ST GW VIP (for ST traffic) or custom VIP. If specified, overrides pl_underlay_sip from DASH_ENI_TABLE
`
Also please update this if this route.underlay_sip field is only applicable for ST and not for PL.

Reply:
Agree @mukeshmv .  Seems like a typo, @theasianpianist , please refer the example section - https://github.com/sonic-net/SONiC/blob/master/doc/dash/dash-sonic-hld.md#363-private-link. Would you also help fix HLD?

Reply:
Ok so just to clarify, `underlay_sip` is never used for PL?

Reply:
yes @theasianpianist that's my understanding as per Sirius. @prsunny can you please re-confirm.

Reply:
@mukeshmv , @theasianpianist , afaik, this can be applicable for PL for some cross-region scenarios. It may not be used for Milestone 1. I would suggest to keep the option available.